config/service-ca/crd_webhooks_patch.yaml

```diff
@@ -0,0 +1,41 @@
+# Add webhooks to the devfile/api CRDs
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: devworkspaces.workspace.devfile.io
+  annotations:
+    service.beta.openshift.io/inject-cabundle: "true"
+spec:
+  conversion:
+    strategy: Webhook
+    webhook:
+      conversionReviewVersions: ["v1"]
+      clientConfig:
+        service:
+          namespace: devworkspace-operator
+          # Note: service name is hard-coded in pkg/webhook/server/server.go
+          name: devworkspace-webhookserver
+          path: /convert
+        # caBundle will be filled by Service CA operator
+        caBundle: Cg==
+---
+# Add webhooks to the devfile/api CRDs
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: devworkspacetemplates.workspace.devfile.io
+  annotations:
+    service.beta.openshift.io/inject-cabundle: "true"
+spec:
+  conversion:
+    strategy: Webhook
+    webhook:
+      conversionReviewVersions: ["v1"]
+      clientConfig:
+        service:
+          namespace: devworkspace-operator
```

Comment:
Should namespace here be parametrized? Seems hard-coded value is also replaced with the right but I would avoid setting any real value to avoid confusion.

Reply:
Good point; updated to `system` which is what we use elsewhere.